tests/acceptance/features/webUIResharing/reshareUsers.feature

```diff
@@ -5,45 +5,58 @@ Feature: Resharing shared files with different permissions
   So that I can control the access on those files/folders by other collaborators
 
   Background:
-    Given these users have been created with default attributes:
+    Given the setting "shareapi_auto_accept_share" of app "core" has been set to "no"
```

Comment:
This whole feature is tagged `skipOnOCIS`
I guess we would need some expected-failures system to be able to run these tests all the time on OCIS, and gradually notice when they start passing.